src/hotspot/share/memory/filemap.hpp

```diff
@@ -457,9 +458,10 @@ class FileMapInfo : public CHeapObj<mtInternal> {
   void  write_header();
   void  write_region(int region, char* base, size_t size,
                      bool read_only, bool allow_exec);
-  void  write_bitmap_region(const CHeapBitMap* ptrmap,
+  char* write_bitmap_region(const CHeapBitMap* ptrmap,
```

Comment:
Can you add a comment here and in MetaspaceShared::write_core_archive_regions() that the returned bitmap is c-heap allocated and needs to be freed?